src/components/ProjectCard/ProjectCard.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./ProjectCard.css";
+
+import { Link } from "react-router-dom";
+
+export default function ProjectCard(props) {
+    const { year, title, img, technologies, bgColor } = props;
```

Comment:
you can use destructuring directly in the function definition  `ProjectCard( { year, title, img, technologies, bgColor } )` usually that is the common practice